src/ui/server/constants/declarations/get-latest-declarations/index.ts

```diff
@@ -0,0 +1,40 @@
+import VERSIONS from '../versions';
+import DECLARATIONS from '../../../content-strings/pages/insurance/declarations';
+import { ObjectType } from '../../../../types';
+
+const { CONFIDENTIALITY, ANTI_BRIBERY, ANTI_BRIBERY_CODE_OF_CONDUCT, ANTI_BRIBERY_EXPORTING_WITH_CODE_OF_CONDUCT, CONFIRMATION_AND_ACKNOWLEDGEMENTS } =
+  DECLARATIONS;
+
+/**
+ * findLatestVersion
+ * Finds the latest content strings for a specified declaration section
+ * @param {ObjectType} declarationContentStrings: Content strings for specified declaration
+ * @param {string} latestVersionNumber: latest version for specified section
+ * @returns {Object} contentStrings: Content strings for specified version of declaration
+ */
+export const findLatestVersion = (declarationContentStrings: ObjectType, latestVersionNumber: string) =>
```

Comment:
Could we rename to `getDeclarationContentStringsByVersionId` so its clearer?

Also could rename `declarationContentStrings` to `contentStrings` and `latestVersionNumber` to `versionNumber` so simpler and more generic